platform/marvell-armhf/rules.mk

```diff
@@ -5,9 +5,9 @@ include $(PLATFORM_PATH)/docker-syncd-mrvl-rpc.mk
 include $(PLATFORM_PATH)/docker-saiserver-mrvl.mk
 include $(PLATFORM_PATH)/libsaithrift-dev.mk
 include $(PLATFORM_PATH)/one-image.mk
-include $(PLATFORM_PATH)/linux-kernel-armhf.mk
```

Comment:
can you remove this file if it is not used.

Reply:
Thanks for pointing it out, I have removed the obsolete files now.